query/iam/iam_deprecated_account.sql

```diff
@@ -2,7 +2,6 @@ with owner_member as (
 select
```

Comment:
Sure,I would love to make the changes

Reply:
@Priyanka-Chatterjee-2000 can you please change it to `disabled_users`, not `disabled_user`? It's a small difference, but it makes more sense to keep the alias as plural since multiple rows can be returned.

Reply:
Hey @Priyanka-Chatterjee-2000 , did you make the alias `disabled_user` instead of `disabled_users` for a specific reason? I think in this case plural works better, since the common table expression (CTE) can return more than 1 user.

Reply:
Yes you are right it can return many users in that case users would be more apt, I will quickly make the changes